src/TodoListItem.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function TodoListItem({todo}) {
+    return (
+        <li>
+            {todo.title}
```

Comment:
There is not need to use todo.title since is already mapped in the TodoList component. 

instead of using todo.list just using **todo** would work